packages/manager-react-components/src/components/navigation/menus/action/action.component.tsx

```diff
@@ -117,6 +117,7 @@ export const ActionMenu: React.FC<ActionMenuProps> = ({
   return (
     <OsdsMenu disabled={disabled || undefined}>
       <OsdsButton
+        className="min-w-[32px]"
```

Comment:
You can use a pre-computed value based on design-tokens here : `min-w-9`

Reply:
Very good idea